src/components/ADempiere/Form/AcctViewer/QueryCriteria/index.vue

```diff
@@ -94,3 +94,9 @@ export default defineComponent({
   }
 })
 </script>
+
+<style>
+.box-card .el-divider--horizontal {
```

Comment:
once this style is loaded it will affect other places where the `el-card` component is used.